src/main/java/com/chopshop166/chopshoplib/outputs/ModSpeedController.java

```diff
@@ -71,6 +76,15 @@ public void addAll(final Collection<? extends Modifier> ms) {
         modifiers.addAll(ms);
     }
 
+    /**
+     * Set the encoder that this speed controller uses.
+     * 
+     * @param enc The encoder.
+     */
+    public void setEncoder(final IEncoder enc) {
```

Comment:
It might be nice to have a constructor that takes an `IEncoder` instead of having to call `setEncoder`.